app/gather/assets/apps/survey/Survey.jsx

```diff
@@ -196,3 +231,5 @@ export default class Survey extends Component {
     )
   }
 }
+
+export default injectIntl(Survey)
```

Comment:
Are you injecting any text? Could you revert this small change? 
`export default class Survey extends Component`